app/src/ui/history/commit-list-item.tsx

```diff
@@ -199,6 +200,17 @@ export class CommitListItem extends React.PureComponent<
       },
     ]
 
+    items.push({
+      label: __DARWIN__
+        ? 'Create Branch From This Commit'
```

Comment:
```suggestion
        ? 'Create Branch from This Commit'
```
Where mac uses title case and `from` is a preposition, it should be lowercase.

Reply:
After testing it out, 
I noticed the revert and cherry-pick menu item  does not have the word 'This' in it.  Just a suggestion, but for consistency we could leave `This` out. 

![image](https://user-images.githubusercontent.com/75402236/117434948-499d4500-aefb-11eb-85c4-bbda4e9d700a.png)